### PR TITLE
Use / prefix for the sql asset paths

### PR DESCRIPTION
**Pull request by LucasRoesler**
**What**
- The union filesystem assumes everything starts with `/`, so when using
  a generated filesystem via vfsgen, we get erros like

  ```sh
  open /iews: file does not exist
  ```

  Adding the slash in the right places allows both the tests and the
  vfsgen to work exactly as expected. This is also hidden from the
  public API, so users should not need to work about it

Signed-off-by: Lucas Roesler <roesler.lucas@gmail.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -51,7 +52,7 @@ setup-env:
 
 .PHONY: .test-ci
 .test-ci:
-	go test -count=1 -cover ./...
+	go test -v -cover ./...
```
Comment (trusch) [inline Makefile:55]: -count=1 causes this to run even when it would otherwise assume that this test is cached. I guess there might be a reason for this

**Review by trusch — DISMISSED** [review]
one comment about the removal of -count=1

File: pkg/queue/postgres/fixtures_test.go [hunk 2]
```diff
@@ -9,3 +15,12 @@ var (
...
 )
+
+
+func verifyLeak(t *testing.T) {
+	_, exists := os.LookupEnv("CI")
+	if exists {
+		return
+	}
+	goleak.VerifyNone(t)
+}
```
Comment (pragmader) [inline pkg/queue/postgres/fixtures_test.go:26]: why do we need it defined twice? there is a copy in `api_request_test.go`.

Comment (LucasRoesler) [reply pkg/queue/postgres/fixtures_test.go:26]: I didn't want to define it as public method

File: Makefile (hunk 1 above)
Comment (LucasRoesler) [reply Makefile:55]: It isn't needed in CI and it was a leftover from my last PR, for example, it wasn't there a last week.

Comment (trusch) [reply Makefile:55]: :+1:

**Review by pragmader — APPROVED** [review]

**Pull request merged**